osidb/models.py

```diff
@@ -820,7 +821,6 @@ def get_queryset(self):
     def fts_search(q):
         """full text search using postgres FTS via django.contrib.postgres"""
         fields_to_search = (
-            "cve_id",
```

Comment:
Because `Affect` does not have `cve_id` attribute any more. It used to have it but it was removed in https://github.com/RedHatProductSecurity/osidb/blob/master/osidb/migrations/0034_remove_affect_cve_id.py - CVE is obviously a flaw attribute not an affect one.

Reply:
OK I see, I somehow missed that this was for affect, the commit message could describe what you wrote

Reply:
I will extend the commit message. Thank you for pointing it out!